Blimp/Blimp.cpp

```diff
@@ -241,6 +223,14 @@ void Blimp::read_AHRS(void)
 {
     // we tell AHRS to skip INS update as we have already done it in fast_loop()
     ahrs.update(true);
+    
+    IGNORE_RETURN(ahrs.get_velocity_NED(vel_ned));
+    IGNORE_RETURN(ahrs.get_relative_position_NED_home(pos_ned));
```

Comment:
need to think a bit about what to do if no position/velocity available

Reply:
I initially tried using the false output from these two functions to trigger a failsafe, but it seems like they only output false when the origin hasn't been set, not when position is lost?

In any case, I'll have a look at what Copter does as you suggested and implement something like it.

Reply:
As discussed, Blimp currently waits for the EKF Failsafe to kick in like Copter does, so I'll mark this as resolved.